cli/storage/storage.py

```diff
@@ -127,11 +122,9 @@ def attach_physical_storage(config, cookies, sys_uuid, partition_uuid, vios_stor
                 f"Attached '{physical_volume_name}' physical volume to the partition from VIOS '{vios_storage_uuid}'")
             break
         except (vios_operation.VIOSError, StorageError, Exception) as e:
-            logger.error(
-                f"failed to attach '{physical_volume_name}' physical storage in VIOS '{vios_storage_uuid}'")
             if index == len(vios_storage_list) - 1:
-                raise e
+                raise StorageError(f"failed to attach '{physical_volume_name}' physical storage in VIOS '{vios_storage_uuid}', error: {e}")
```

Comment:
guess we can rephrase to `failed to attach physical storage: {'{physical_volume_name}' to VIOS '{vios_storage_uuid}'`